content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,43 +2,68 @@
 title: Google Cloud Run
 kind: documentation
 further_reading:
-    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
-      tag: 'Blog'
-      text: 'Collect traces, logs, and custom metrics from GCR services'
+
+- link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+  tag: 'Blog'
+  text: 'Collect traces, logs, and custom metrics from GCR services'
+
 ---
 
 ## Overview
-Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution, now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
+
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog
+provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution,
+now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom
+metrics, and direct log collection.
 
   <div class="alert alert-warning">This feature is in public beta. You can provide feedback through a <a href="https://forms.gle/HSiDGnTPvDvbzDAQA">feedback form</a>, or through your standard support channels. During the beta period, Cloud Run monitoring and APM tracing are available without a direct cost. Existing APM customers may incur increased span ingestion and volume costs. </div>
 
-## Tracing and custom metrics
-### Build your container
+## Getting started
+
+### Prerequisites
+
+Make sure you have a [Datadog API Key][10] and are using a programming
+language [supported by a Datadog tracing library][2].
+
+### Deploy a sample application in one click
+
+To deploy a sample application without the need to follow the rest of the guide, you can use
+the [Cloud Run Button](https://github.com/GoogleCloudPlatform/cloud-run-button)
```

Comment:
Is it necessary to have the link for the "Cloud Run Button"? We can probably link it in the example repo's readme instead. So when user get to the readme and wonder what that big blue button is, they see a link to explain it.